www/proxy-sso/+POST_DEINSTALL.post

```diff
@@ -0,0 +1,5 @@
+rm /usr/local/etc/squid/pre-auth/20-negotiate.auth.conf
```

Comment:
I don't have time for a full review yet, but this needs "rm -f" in case the file is not there

I have this on my TODO to provide a proper revoke of templates on deinstall for automation: https://github.com/opnsense/plugins/issues/46